### PR TITLE
transcript and comment endpoints functional

### PR DESCRIPTION
**Pull request by edward2018211**
Endpoints for comments ('/score/comments') and transcript ('/score/transcript') are now functional. When a POST request is sent with the data.json as request file, the corresponding content will be sent back in JSON format.

### CODE REVIEW

**Comment by pandevim** [comment]
# First ever feature PR!!
Awesome

**Comment by edward2018211** [comment]
Just made the requested changes. Regarding naming branches, I will make sure to do that going forward. All other points should be addressed.

File: api.py [hunk 1]
```diff
@@ -1,53 +1,22 @@
...
-  		# print("An exception occurred")
-  		return(None)
-
 @app.route('/score/transcript', methods=['POST'])
 def transcript():
 	response = request.json
 	video_id = response["videoID"]
-	transcript_list = get_transcript(video_id)
-	return jsonify(results = transcript_list)
+	transcript_list = get_transcript.get_transcript(video_id)
```
Comment (pandevim) [inline api.py:10]:
```python
get_transcript.get_transcript(video_id)
```
seems unnecessary right?

---
maybe on #L2 we can
```python
from service.transcript import get_transcripts
```
and use it as 
```python
transcript_list = get_transcripts(video_id)
```

File: api.py [hunk 2]
```diff
@@ -1,53 +1,22 @@
...
+	return jsonify(results=transcript_list)
 
 @app.route('/score/comments', methods=['POST'])
 def comments():
 	response = request.json
 	video_id = response["videoID"]
 	kTop = response["kTopComments"]
-	comments_list = get_comments(video_id, kTop)
-	return jsonify(results = comments_list)
+	comments_list = get_comments.get_comments(video_id, kTop)
```
Comment (pandevim) [inline api.py:18]: same changes as #L10

**Comment by edward2018211** [comment]
Sounds good. I just pushed those suggestions onto the branch.

**Pull request merged**